app/coffee/Features/User/UserController.coffee

```diff
@@ -44,6 +44,7 @@ module.exports = UserController =
 
 	updateUserSettings : (req, res)->
 		user_id = AuthenticationController.getLoggedInUserId(req)
+		usingExternalAuth = !!(settings.ldap? or settings.saml?)
```

Comment:
Possibly not, just forcefully coercing to boolean. Without the `!!`, `usingExternalAuth` ends up as whatever the value of `settings.saml` or `settings.ldap` is. It'll be truthy for sure, but doesn't feel as clean.